packages/@core.keystone/errors.js

```diff
@@ -124,6 +124,7 @@ class GQLError extends ApolloError {
         const message = template(fields.message)(fields.messageInterpolation)
         extensions.message = message
         if (context) {
+            // todo use i18n from apps/condo/domains/common/utils/localesLoader.js
```

Comment:
please add ticket for TODO
`// TODO(DOMA-xxxx): ...`

Reply:
This is not an urgent task. The next one who will edits this code will see this comment and maybe simplifies the code.